docs/api.md

```diff
@@ -1010,7 +1010,8 @@ Generate embeddings from a model
 ### Parameters
 
 - `model`: name of model to generate embeddings from
-- `prompt`: text to generate embeddings for
+- `prompt`: string to generate the embedding for
+- `prompts`: array of strings to generate a batch of embeddings for
```

Comment:
it's either `prompt` or `prompts` right? if given both, only `prompt` is processed? (do we want to make it explicit here?)